openmetadata-ui/src/main/resources/ui/src/pages/teams/AddUsersModalV1.tsx

```diff
@@ -88,7 +88,7 @@ const AddUsersModalV1 = ({
       });
   };
 
-  const searchUsers = (text: string, page: number) => {
+  const searchUsers = (text: string, page = 1) => {
```

Comment:
@ShaileshParmar11 , Why we are making this change?

Reply:
As it is not picking reset value, so I have updated the logic, if we want to reset the size we will not send any number but if we want to update API call based on page number we will pass page value!